sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -1732,44 +1801,348 @@ block_store(T *ptr, size_t byte_offset, simd_view<Toffset, RegionTy> vals,
   block_store<T, N>(ptr, byte_offset, vals.read(), pred, props);
 }
 
-/// @} sycl_esimd_memory_block
+/// Each of the following block store functions stores a contiguous memory block
+/// from the address referenced by accessor 'acc', or from 'acc + byte_offset',
```

Comment:
These 2 lines may need some update. The 'from' seems a bit confusing here.
```suggestion
/// Each of the following block_store functions stores the vector 'vals' to a contiguous memory
/// block at the address referenced by accessor 'acc', or 'acc + byte_offset',
```

Reply:
fixed, thx